### ISSUE TITLE
 Upgraded the Default PKCS12 Encryption Algorithms JDK-8153005

### ISSUE THREAD

**Issue by ed03875** [adoptium/adoptium-support#378]
## Question

Why were the Oracle upgraded default PKCS12 Encryption Algorithms not in the AdoptOpenJDK release  8u302 ?  I see that they are there in the Oracle 8u301 release which I thought was the basis for the AdoptOpenJDK release. .

## Context

Java version:
openjdk version "1.8.0_302"
OpenJDK Runtime Environment (Temurin)(build 1.8.0_302-b08)
OpenJDK 64-Bit Server VM (Temurin)(build 25.302-b08, mixed mode)Paste the entire output of java -version.

Your operating system and platform:
Windows 64 bit

**Comment by jerboaa** [comment]
> https://bugs.openjdk.java.net/browse/JDK-8258401 is the backport for reference. Can you show us some output demonstrating the out of date algos?

No, it's not. This is Oracle JDK 8u (which is private). Temurin doesn't use Oracle JDK as base, but **OpenJDK 8u**. An OpenJDK backport would have "Fix Version: openjdk8u<update-ver>". The above bug has `8u301`. So no, this hasn't been backported to OpenJDK yet. Feel free to contribute a patch upstream!

**Comment by karianna** [comment]
Aha - I got confused again, thanks, Severin!
